src/test/java/org/opentripplanner/middleware/docs/SwaggerTest.java

```diff
@@ -0,0 +1,45 @@
+package org.opentripplanner.middleware.docs;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.middleware.OtpMiddlewareTest;
+import org.opentripplanner.middleware.utils.YamlUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.http.HttpResponse;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.opentripplanner.middleware.docs.PublicApiDocGenerator.getSwaggerDocs;
+import static org.opentripplanner.middleware.docs.VersionControlledSwaggerUpdater.SWAGGER_CONTROLLED_FILE;
+
+/**
+ * Contains test to verify that the swagger docs are generated correctly.
```

Comment:
addition to comment: `... and up to date.`

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/59/commits/b327a799b261096bd67fcaf0d680d92b1090fc2a